src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -604,9 +604,18 @@ void ClusterResourceScheduler::InitLocalResources(const NodeResources &node_reso
 
   for (size_t i = 0; i < PredefinedResources_MAX; i++) {
     if (node_resources.predefined_resources[i].total > 0) {
+      bool is_unit_instance = false;
+      auto find_unit = UnitInstanceResources.find(i);
```

Comment:
It's not a good practice to define a global static variable in header file. I'd like to move `UnitInstanceResources` to other place.